spec/registry.xml

```diff
@@ -0,0 +1,147 @@
+      <!-- The representation of the day. -->
+      <option name="day" values="numeric 2-digit"/>
+      <!-- The representation of the hour. -->
+      <option name="hour" values="numeric 2-digit"/>
+      <!-- The representation of the minute. -->
+      <option name="minute" values="numeric 2-digit"/>
+      <!-- The representation of the second. -->
+      <option name="second" values="numeric 2-digit"/>
+      <!-- The number of digits used to represent fractions of a second
+        (any additional digits are truncated). -->
+      <option name="fractionalSecondDigits" values="1 2 3"/>
+      <!-- The localized representation of the time zone name. -->
+      <option name="timeZoneName" values="long short shortOffset longOffset shortGeneric longGeneric"/>
+    </formatSignature>
+
+  </function>
+
+  <function name="number">
+    <!-- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat -->
+    <description>Locale-sensitive number formatting</description>
+
+    <formatSignature>
+      <input pattern="anyNumber"/>
+      <!-- Only used when notation is "compact". -->
+      <option name="compactDisplay" values="short long" default="short"/>
+      <!-- The currency to use in currency formatting.
+        Possible values are the ISO 4217 currency codes, such as "USD" for the US dollar,
+        "EUR" for the euro, or "CNY" for the Chinese RMB — see the
+        Current currency & funds code list
+        (https://www.six-group.com/en/products-services/financial-information/data-standards.html#scrollTo=currency-codes).
+        There is no default value; if the style is "currency", the currency property must be provided.
+      -->
+      <option name="currency" pattern="currencyCode"/>
+      <!-- How to display the currency in currency formatting. -->
+      <option name="currencyDisplay" values="symbol narrowSymbol code name" default="symbol"/>
+      <!-- In many locales, accounting format means to wrap the number with parentheses
+        instead of appending a minus sign. You can enable this formatting by setting the
+        currencySign option to "accounting".
+      -->
+      <option name="currencySign" values="accounting standard" default="standard"/>
+      <!-- The formatting that should be displayed for the number. -->
+      <option name="notation" values="standard scientific engineering compact" default="standard"/>
+      <!-- Numbering system to use. -->
+      <option name="numberingSystem" values="arab arabext bali beng deva fullwide gujr guru hanidec khmr knda laoo latn limb mlym mong mymr orya tamldec telu thai tibt"/>
+      <!-- When to display the sign for the number. -->
+      <!-- "negative" value is Experimental. -->
+      <option name="signDisplay" values="auto always exceptZero never" default="auto"/>
+      <!-- The formatting style to use. -->
+      <option name="style" values="decimal currency percent unit" default="decimal"/>
```

Comment:
To consider: If we were to leave out `currency` and `unit` formatting from the default, then we wouldn't need to say how compound units need to work.